source/Ship.cpp

```diff
@@ -3111,9 +3111,9 @@ void Ship::ApplyForce(const Point &force, bool gravitational)
 		return;
 	
 	// Reduce acceleration of small ships and increase acceleration of large
-	// ones by having 30% of the force be based on a fixed mass of 400, i.e. the
-	// mass of a typical light warship:
-	acceleration += force * (.3 / 400. + .7 / currentMass);
+	// ones by having 20% of the force be based on a fixed mass of 500, about the
+	// mass of a human light warship:
+	acceleration += force * (.2 / 500. + .8 / currentMass);
```

Comment:
I'm tempted to simply remove having a fraction of hit force be based on a fixed mass altogether.

Given the values I'm currently using, the lightest ship in the game, the Surveillance Drone with a mass of 63, feels 18% less hit force because of this equation. Then as you get closer to 500 mass, the % reduction in felt hit force gets smaller. On the opposite end, once you get to 1000 tons of mass (about the mass of a Mule or Behemoth under this PR), you're feeling +20% hit force, and it only increases from there. The Protector, the lightest heavy warship at 1400 tons, is feeling +36% hit force. The Cruiser at 2600 tons is feeling +84% hit force. The KIV at about 4000 tons is feeling +140% hit force. The Emerald Sword at 4350 is feeling +154% hit force. Then the Heron at 28600 tons is feeling +1124% hit force.

So really, all this does is reduce the hit force on smaller ships by a negligible amount while MASSIVELY increasing the hit force on larger ships. So either we could just outright remove this weighting (with #5978 we could keep the ability to have it and just weight the left side 0), or massively up the fixed mass. If we increased the fixed mass from 500 to 2000, the Surveillance Drone would go from feeling 18% less force to 19.5% less force, so on the low end it really doesn't change anything, but on the higher end a 4000 ton ships will only experience +20% hit force, and the Heron will feel +266% hit force. The Heron, or any ship that is wildly more massive than anything else, will always be an outlier under a system like this, but a fixed mass of 2000 makes more sense to me than 500, as it's the balance point where the smallest and largest ships feel a similar change in hit force (about -20% on the low end and about +20% on the high end).

Reply:
I rather like the idea of just removing the modifier entirely. Having the hit force modifier makes sense if the assumption is that all the ships are going to be clustered in the middle and one is trying to balance all these similar ships to be affected fairly similarly.

Honestly, though, a) that's boring, and b) unrealistic. If a surveillance drone gets whacked by the force of a heavy rocket (and by some miracle survives), I *expect* it to get booted across the system. This is not a problem as far as I'm concerned. On the other hand, when a world-ship gets hit by a heavy rocket, it should barely register at all.

Reply:
I'm all  for removing the fractional application of force but just to put an alternative on the table, we could fix the formula. It's incorrect.

Currently:  acceleration += force * (.3 / 400. + .7 / currentMass);

Should be: acceleration += force / (.3 * 400 + .7 * currentMass);

I suspect an accedental transposing of * and / way back and not until fairly recently has there been a ship with enough mass to really highlight the fault.

Run the numbers and see what you think.

Reply:
> It's incorrect.

No, the formula is working as intended.
![image](https://user-images.githubusercontent.com/17688683/120839704-2081cb80-c51e-11eb-89c2-dce0c78dba60.png)

Reply:
For comparison, I have removed the hit force adjustment from Revamp entirely. So anyone that wants to see how impacts feel in various ships without any formulas artificially changing things can do so.

Reply:
I'm aware of the code comment and it is somewhat ambiguous but I think if you run some numbers for both you'll see.